client/src/components/Atoms/Form/_helpers.tsx

```diff
@@ -10,7 +10,10 @@ export const generateOptions = (count: number): JSX.Element[] => {
 }
 
 export const ErrorLabel = styled.p`
-  color: #ff0000;
+  color: #a82a2a;
+`
+export const SuccessLabel = styled.p`
+  color: #0a6640;
```

Comment:
Oh I meant to say on the previous review - i realized recently you can use the `Colors` object from Blueprint. So this could be `Colors.GREEN2` or something (not sure exactly which color to use). doesn't need to be changed necessarily but nice to know about i think